### DISCUSSION TITLE
My main account is locked due to 2FA without having recovery codes, how can I recover it?

### DISCUSSION THREAD

**Discussion by AhmedThresh** [community/community#103766] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Hey folks,
My main account named @AhmedGrati  is locked due to 2fa. I actually lost my 2FA device, and I don't have any access to the recovery codes. 
I have access to all accounts linked to the account, and I have access to the account from my local machine. I submitted a ticket 3 days ago, but I haven't gotten a response yet.
I want to know whether I can recover my account since I have a lot of open-source work that I can't lose, and how I should proceed in the current situation.

**Comment by DJLink** [comment]
Unfortunately if you lost access to all your 2FA options (codes, passkeys, SMS etc) they can't recover your account. 

Are you sure you don't have still access to any of the methods, maybe an old device with the credentials still on, an ssh key etc?

There is an option to reset the 2FA credentials on the login page (see gif) but you need one of things indicated in that page. 
you can 

https://docs.github.com/en/authentication/securing-your-account-with-two-factor-authentication-2fa/recovering-your-account-if-you-lose-your-2fa-credentials


![2024-02-04 13-40-55](https://github.com/community/community/assets/423769/5d30bfe5-8f42-42bb-a5d3-26526a59d915)

**Reply by AhmedThresh** [reply]
Unfortunately not, the only access I have rn is to my emails and account in my local machine. 
Can't they disable the 2FA of the account for me?

**Reply by DJLink** [reply]
> Can't they disable the 2FA of the account for me?


According to the link I sent above, what I understand is they can only do that if you have access to at least 1 of them.
But contacting support like you did is the best choice here, they might take a few days like mentioned on the help page.

![image](https://github.com/community/community/assets/423769/61c8f9e9-9c72-4378-baf4-27e27915aade)

**Reply by AhmedThresh** [reply]
Thanks @DJLink for your efforts! Let's wait for one of the support team members to catch up on this.

**Comment by AhmedThresh** [comment]
@mishmanners Can you please help me with this? I'm feeling stuck and I don't know what should I do or who should I contact!

**Reply by mishmanners** [reply, marked as answer]
Looks like you'll need to open a support ticket:

1. Go to the portal: https://support.github.com/request/landing
2. Click "can't sign in"
3. Type in your email address (preferably the one associated with your GitHub account)
4. Select "Account locked..."
5. Verify
6. Click "Submit"
7. Enter any other information and submit the ticket


The support team should be able to handle this. Hopefully that helps.

**Reply by AhmedThresh** [reply]
@mishmanners Thanks, I opened a ticket, and I hope that they'll resolve this soon!

**Reply by AhmedThresh** [reply]
@mishmanners do you have any idea how much it takes to process these kinds of tickets? I opened a ticket last week and no response yet...

**Reply by mishmanners** [reply]
Our support team works hard to get back to every single ticket. I am not sure how many tickets are currently in the queue so I can't say how long your ticket will take to process sorry.

**Reply by RedouaneBALI** [reply]
> Looks like you'll need to open a support ticket:
> 
> 1. Go to the portal: https://support.github.com/request/landing
> 2. Click "can't sign in"
> 3. Type in your email address (preferably the one associated with your GitHub account)
> 4. Select "Account locked..."
> 5. Verify
> 6. Click "Submit"
> 7. Enter any other information and submit the ticket
> 
> The support team should be able to handle this. Hopefully that helps.

Hello, the chatbot looks bugged. Any idea to solve this issue ? I also opened a ticket weeks ago but still not answer :'(

<img width="870" alt="image" src="https://github.com/user-attachments/assets/97e29b6d-456c-4fab-80e8-f0d5846b56b2">

**Reply by upsidecode** [reply]
> Our support team works hard to get back to every single ticket. I am not sure how many tickets are currently in the queue so I can't say how long your ticket will take to process sorry.

If your team "works hard" and can't handle every single it. Then, why your team make login flow like this? mandatory requiring 2fa? what a shame

**Comment by lucasccdias2** [comment]
I am having the same issue right now. How this ended, @AhmedThresh ?

**Reply by TessaIO** [reply]
The answer by the support is that: I cannot get back my account if I don't have a way to prove my identity cryptographically (SSH key or smth)

**Reply by febrian-hub** [reply]
how about now? did you get your account back?

**Reply by TessaIO** [reply]
no

**Comment by David-OConnor** [comment]
I dread the day I lose my phone, or it breaks.

**Comment by dawitesfa** [comment]
Do not enable 2fa

**Reply by RedouaneBALI** [reply]
it became mandatory...

**Comment by mikel-codes** [comment]
Github, has become a thorn in the flesh, I opened account 2fa locked out ticket with github and after 2months of waiting , see their reaction
For over a period of 2months they did not reply and when they did all they replied was their their previous responses are the same and all of that is a lie because I never had any previous other than waiting on the queue for them 

My work is urgent to me , i have appealed in every way possible and told them the reason behind my lockout was due to a failed SSD which removed all i saved earlier and used for github

Here is my github support ticket 
**Request to Retrieve Repositories from my Locked Account due to 2FA: @iamunadike #3595101**

and the worst aspect is no relevant response has been given to me since , because I only got messages of **_I am on the waiting queue_**
I submitted literally everything i had , snapshots of previous emails CI/CD pipelines, Personal Access token I had one time tied to the account and my email and password

All the feedback is a lie here 
<img width="1421" height="475" alt="Screenshot 2025-09-24 at 3 44 18 AM" src="https://github.com/user-attachments/assets/3f077674-5106-4046-894b-5d66a7a9f415" />
I never had any responses, this was the only response after a month and 3weeks of waiting , and the worst aspect the person went ahead to close the ticket without any form of remorse

I had asked github to help me download my repositories on their platform and send to my email or something as a last option , but it seems the platform is something of dark and sinister or strangely evil because they aka github don't have any right to hold any developers code , its almost like a bank saying you can not remove your money from your account assuming you lost some passkey to authenticate , but definitely the bank must give you your money if you prove with ownership you own that account

Github return my work to me , I want all my work from your platform and its a shame github has stooped this low, this is why many developer and people are not abandoning github for safer and more efficient git platforms
